uaclient/entitlements/repo.py

```diff
@@ -58,7 +58,20 @@ def messaging(
     @property
     def packages(self) -> "List[str]":
         """debs to install on enablement"""
-        return []
+        packages = []
+
+        if self.cfg:
```

Comment:
We will never have an entitlement with an uninitialized self.cfg because [base.py:Entitlement.__init__](https://github.com/canonical/ubuntu-advantage-client/blob/master/uaclient/entitlements/base.py#L68-L69) will set self.cfg = UAClientConfig() if the cfg param is not provided.

So this if can be dropped.